common/conf.example

```diff
@@ -6,12 +6,19 @@ adminMail=admin@example.com
 basePath=/srv
 
 # Site name = user name that executes the site.
-userNames="org1 org2"
+userNames=""
+
+# Use letsencrypt for HTTPS? If so, DNS records need to be accurate before installing!
```

Comment:
Well, that's what I did, perhaps I can express it more succintly: `s/accurate/point to the server running this script/`.